bundles/org.openhab.binding.remoteopenhab/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -18,14 +18,32 @@
 				<required>true</required>
 			</parameter>
 
+			<parameter name="useHttps" type="boolean">
```

Comment:
The `required` tag is deprecated.
```suggestion
			<parameter name="useHttps" type="boolean" required="true">
```

Reply:
I did not know. Where is it mentioned ?

Reply:
It is only for channel parameter ? Not for thing type parameter ?

Reply:
Changed

Reply:
It's deprecated for `config-description`. AFAIK these can be used for both channel and thing type parameters. That'd imply it's deprecated for both. https://www.openhab.org/docs/developer/bindings/config-xml.html#xml-structure-for-configuration-descriptions

Reply:
In this case, it should be updated for things too, I will submit a fix later.